src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color,int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String toString() {
+        System.out.println("Circle{" + "radius=" + radius
+                + ", color='" + getColor() + '\'' + ", area=" + getArea() + '}');
+        return "";
```

Comment:
```suggestion
        return "Circle{" + "radius=" + radius
                + ", color='" + getColor() + '\'' + ", area=" + getArea() + '}';
```

Reply:
behaviour - we can obtain the area of any figure and are able to draw it. To 'draw' means to print out all information about a figure using System.out.println() (`you shouldn't override toString() method for this)`.